Tasks/XamariniOS/xamariniOS.js

```diff
@@ -30,21 +30,40 @@ if (!fs.existsSync(mdtoolPath)) {
     tl.exit(1);
 }
 
-// Prepare build command line
-var mdtoolRunner = new tl.ToolRunner(mdtoolPath);
-mdtoolRunner.arg('--verbose');
-mdtoolRunner.arg('build');
-mdtoolRunner.arg('--configuration:' + configuration + '|' + device);
-mdtoolRunner.arg(solutionPath);
-
-// Execute build
-mdtoolRunner.exec()
-.then(function (code) {
-    // Executed successfully
-    tl.exit(code);
-})
-.fail(function (err) {
+// Find location of nuget
+var nugetPath = tl.which('nuget');
+if (!nugetPath) {
+    tl.error('nuget was not found in the path.');
```

Comment:
Executing `which nuget` on my build agent reveals the path `/usr/local/bin/nuget`. Why would I be getting the error message `nuget was not found in the path`? Is something other than the standard nuget path being passed?